CHANGELOG.md

```diff
@@ -4,6 +4,89 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [0.2.5]
+### Added
+- Test multiple collaborators on a single share https://github.com/owncloud/phoenix/pull/2081
+- Run tests on internet explorer https://github.com/owncloud/phoenix/pull/2082
+- Draw.io app integration https://github.com/owncloud/phoenix/pull/2083
+- Await trashbin deletion requests in acceptance tests https://github.com/owncloud/phoenix/pull/2093
+- Check that required permissions are displayed for files https://github.com/owncloud/phoenix/pull/2098
+- Logout step for acceptance tests https://github.com/owncloud/phoenix/pull/2100
+- New file menu entries for different file types https://github.com/owncloud/phoenix/pull/2111
+- Drone starlark https://github.com/owncloud/phoenix/pull/2112
+- Properly reset capabilities on logout https://github.com/owncloud/phoenix/pull/2116
```

Comment:
that should belong under "Fixed" ?

Reply:
Will move it there. I originally thought it fits here because the reset wasn't there at all. But yeah, make sense to have it in Fixed. Thanks